ramls/lost-item-fee-policy.json

```diff
@@ -22,6 +22,16 @@
       "type": "object",
       "$ref": "period.json"
     },
+    "recalledItemAgedLostOverdue": {
```

Comment:
We have to update minor version of the API that uses this schema in both RAML and Module descriptor files.

Reply:
Ah, of course.  I'll adjust.

Reply:
I tried doing this-I'm not sure I did it right.  I'm noticing that the version in the module descriptor does not match the one in NEWS.md...aren't they supposed to?

Reply:
I think this are different versions. The one in the NEWS file are the module versions (from pom) but in module descriptor and in RAML it is API versions.